dist/src/main/java/io/camunda/application/commons/backup/ElasticsearchOperateBackupRepository.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application.commons.backup;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.webapps.backup.BackupRepository;
+import io.camunda.webapps.backup.repository.BackupRepositoryProps;
+import io.camunda.webapps.backup.repository.WebappsSnapshotNameProvider;
+import io.camunda.webapps.backup.repository.elasticsearch.ElasticsearchBackupRepository;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@ConditionalOnProperty(
+    prefix = "camunda.database",
+    name = "type",
+    havingValue = "elasticsearch",
+    matchIfMissing = true)
+@Configuration
+@Profile("operate")
```

Comment:
:question: Why not for standalone? Is it because we use Operate's for both when not standalone? Shouldn't we name it `ElasticsearchBackupRepository` then?

Reply:
Yes that's the reason. I will rename it.